API/SSW.Rewards.Application/User/Queries/GetCurrentUser/AutoMapperProfile.cs

```diff
@@ -8,19 +8,11 @@ namespace SSW.Rewards.Application.User.Queries.GetCurrentUser
 {
     public class AutoMapperProfile : Profile
     {
-        public AutoMapperProfile(ILogger<AutoMapperProfile> logger)

```

Comment:
Ooops, can't do this. Must have a parameterless constructor on the AutoMapperProfiles